http://pkgs.fedoraproject.org/gitweb/?p=xscreensaver.git;a=commit;h=88b852d368c4c5ec8b798913e4856e9aed9ed936

sonar-icmp.c:594:55: error: ‘AF_LINK’ undeclared (first use in this function)

--- hacks/glx/sonar-icmp.c
+++ hacks/glx/sonar-icmp.c
@@ -591,7 +591,9 @@
                 fprintf (stderr, "%s:     if: %4s: %s\n", progname,
                          ifa->ifa_name,
                          (ifa->ifa_addr->sa_family == AF_UNIX  ? "local" :
+#ifdef AF_LINK
                           ifa->ifa_addr->sa_family == AF_LINK  ? "link"  :
+#endif
                           ifa->ifa_addr->sa_family == AF_INET6 ? "ipv6"  :
                           "other"));
               continue;
